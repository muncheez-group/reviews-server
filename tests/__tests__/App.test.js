import Enzyme, { shallow, render, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react';
import ReactDOM from 'react-dom';
import App from '../../client/index.jsx'

Enzyme.configure({ adapter: new Adapter() });

describe('App Parent Component', () => {
  const wrapper = shallow(
    <App/>
  );

  test('should render App', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('should have 2 child components', () => {
    console.log(wrapper.children())
    expect(wrapper.children().length).toBe(2);
  });

});
