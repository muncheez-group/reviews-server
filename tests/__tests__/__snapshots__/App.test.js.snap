// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Parent Component should render App 1`] = `
<div>
  <Description />
  <Reviews />
</div>
`;
